pkger/testdata/dashboard_heatmap.yml

```diff
@@ -38,17 +38,17 @@ spec:
             - hex: "#fbb61a"
             - hex: "#f4df53"
           axes:
-            - name: x
-              label: x_label
-              prefix: x_prefix
-              suffix: x_suffix
+            - name: "x"
+              label: "x_label"
+              prefix: "x_prefix"
+              suffix: "x_suffix"
               domain:
                 - 0
                 - 10
-            - name: y
-              label: y_label
-              prefix: y_prefix
-              suffix: y_suffix
+            - name: "y"
```

Comment:
this should be the only change you need out of all these changes 🤔, we're the tests failing here?

Reply:
they were, i'll revert it and test again to get the exact error as i can't remember what the error was

Reply:
glad to hear the tests failing though 👍

Reply:
Comment on x makes sense. tests pass without the quotes now, but I'll leave them in. i quoted both for uniformity's sake.